packages/server/jiraImagesHandler.ts

```diff
@@ -1,20 +1,25 @@
 import {promises as fsp} from 'fs'
-import mime from 'mime-types'
 import path from 'path'
 import {HttpRequest, HttpResponse} from 'uWebSockets.js'
 import jiraPlaceholder from '../../static/images/illustrations/imageNotFound.png'
 import sleep from '../client/utils/sleep'
 import uWSAsyncHandler from './graphql/uWSAsyncHandler'
 import getRedis from './utils/getRedis'
 
-const getImageFromCache = async (fileName: string, tryAgain: boolean) => {
+const getImageFromCache = async (imgUrlHash: string, tryAgain: boolean) => {
   const redis = getRedis()
-  const imageBuffer = await redis.getBuffer(fileName)
+  const [[imageErr, imageBuffer], [contentTypeErr, contentType]] = await redis
+    .multi()
```

Comment:
-1 since we always fetch these in tandem & they both share the same expiration, we can save some lookup time & memory by using a hash data type in redis.
Not strictly necessary, but it could clean things up a bit & set a a good example for how to use hashes moving forward https://redis.io/commands#hash

Reply:
> storing data

yep! this looks great. a pipeline with `pexpire` is the right call

> read the data

you could use `hgetallBuffer` and then call `imageBuffer.contentType.toString()` if you don't want to use a pipeline. it's probably slightly more efficient (i don't think redis internally caches recent reads like that), but tbh i don't think the performance is a big deal either way.

The command exists in redis, but it's not in the typescript file! it's a secret 🤭

Reply:
thanks, @mattkrick. i've updated the PR so I'd appreciate it if you could take a look again 🙏🏻